src/casp/anndata_to_avro.py

```diff
@@ -28,7 +28,7 @@ def current_milli_time():
     return round(time.time() * 1000)
 
 
-def write_avro(generator, parsed_schema, filename, progress_batch_size=1000):
+def write_avro(generator, parsed_schema, filename, progress_batch_size=10000):
```

Comment:
just curious — why was this upped?

Reply:
I apparently had an issue building a native code piece of a numerical library for one of my Python versions, which caused the library to fall back to a super slow pure Python implementation. At some point I reinstalled this Python version but this time the library successfully. These operations got ~10x faster and I was able to raise the size of batches between progress updates.